lucille-examples/lucille-simple-csv-solr-example/conf/simple-csv-solr-example.conf

```diff
@@ -0,0 +1,30 @@
+# this example illustrates how we can use Lucille to take something as simple as a CSV and index it into a search engine like Solr

```

Comment:
this example illustrates how Lucille can handle a simple use case like indexing the contents of a CSV file into Solr

Reply:
Done